plutus-tx-plugin/plutus-tx-plugin.cabal

```diff
@@ -100,7 +104,7 @@ library
     build-depends: ghc >=9.2
```

Comment:
This `build-depends: ghc >= 9.2` should be reduced to `build-depends: ghc` because we guard already the version using `ghc-version-support`